subsys/bluetooth/mesh/vnd/Kconfig

```
@@ -31,4 +31,41 @@ config BT_MESH_SILVAIR_ENOCEAN_AUTO_COMMISSION
 
 endif
 
+config BT_MESH_DM_CLI
+	bool "Distance Measurement Client"
+	select BT_MESH_VENDOR_MODELS
+	depends on DM_MODULE
+	help
+	  Enable Distance Measurement Client model.
+
+config BT_MESH_DM_SRV
+	bool "Distance Measurement Server"
+	select BT_MESH_VENDOR_MODELS
+	depends on DM_MODULE
+	help
+	  Enable Distance Measurement Server model.
+
+if BT_MESH_DM_SRV
+
+config BT_MESH_DM_SRV_DEFAULT_TTL
+	int "Default distance measurement Time to Live"
+	default 0
+	help
+	  Default Time to Live for distance measurement attempt.
```

```suggestion
	  Default Time to Live for distance measurement attempt. The value of '1' is an invalid value and shall not be used.
```

Can you also add a compile time assertion in dm_srv.c to check that CONFIG_BT_MESH_DM_SRV_DEFAULT_TTL is not set to `1`?